main.py

```diff
@@ -35,6 +42,23 @@ def setup(self):
 
         So, if you have any setup routine, insert it here.
         """
+        # Validate Spec
+        self.napps_manager = NAppsManager(self.controller)
+        enabled_path = str(self.napps_manager._enabled_path)
+        if enabled_path[:2] == "//":
+            enabled_path = enabled_path[1:]
+        yml_file = enabled_path + "/.installed/kytos/mef_eline/openapi.yml"
+        spec_dict, spec_url = read_from_filename(yml_file)
+        try:
+            spec_error = validate_spec(spec_dict)
+        except ValueError as err:
+            raise BadRequest(err) from err
+        else:
+            if spec_error:
+                log.debug("spec_url %s", spec_url)
+                raise BadRequest(spec_error) from spec_error
```

Comment:
again, not sure if we should raise the BadRequest here (same as previous comment)